request.js

```diff
@@ -0,0 +1,47 @@
+//var rp = require('request-promise');
+import requestPromise from 'request-promise';
+
+class Request {
+    constructor(method, url) {
```

Comment:
You constructor is useless if you decide to create a full static class
Furthermore, if your whole class is static, class is useless

Why don't you instantiate a request and perform it ?

Reply:
You're right, what is the best solution ? A constructor that instantiate the class + perform it after ? 
or just a method like performRequest?